cirq-google/cirq_google/devices/coupler.py

```diff
@@ -0,0 +1,90 @@
+# Copyright 2024 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Any, Dict, Optional, Tuple, TYPE_CHECKING
+
+from cirq import ops, protocols
+
+if TYPE_CHECKING:
+    import cirq
+
+
+class Coupler(ops.Qid):
+    """A Qid representing the coupler between two qubits."""
+
+    _qubit1: ops.Qid
+    _qubit2: ops.Qid
```

Comment:
nit: should we zero-index?
```suggestion
    _qubit0: ops.Qid
    _qubit1: ops.Qid
```

Reply:
Changed to zero indexing.